src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,51 @@
+package duke.ui;
+
+
+/**
+ * The Class Ui provides a basic user interface.
+ */
+public class Ui {
+    
+    /** The logo. */
+    protected String logo = " ____        _        \n"
```

Comment:
Perhaps you want to make this into a FINAL variable? Since the logo will not change!